mixer/cmd/build.go

```diff
@@ -112,12 +112,35 @@ var buildBundlesCmd = &cobra.Command{
 	},
 }
 
+var buildUpstreamFormatCmd = &cobra.Command{
+	Use:   "upstream-format",
+	Short: "Use to create the necessary builds to cross an upstream format",
+	Long:  `Use to create the necessary builds to cross an upstream format`,
+	Run: func(cmd *cobra.Command, args []string) {
+		b, err := builder.NewFromConfig(config)
+		if err != nil {
+			fail(err)
+		}
+		setWorkers(b)
+		err = b.BuildUpdate(buildFlags.prefix, buildFlags.minVersion, buildFlags.format, buildFlags.noSigning, !buildFlags.noPublish, buildFlags.keepChroot)
+		if err != nil {
+			failf("Couldn't build update: %s", err)
+		}
```

Comment:
Erm, wait, this is just doing a regular update build. This command needs to do the whole `b.RunCommandInContainer(cmd)` thing for both of the sub commands, right?

```go
	cmd := strings.Split("mixer build upstream-format new", " ")
	b.RunCommandInContainer(cmd)
	cmd = strings.Split("mixer build upstream-format old", " ")
	b.RuNCommandInContainer(cmd)
```

Reply:
Wait, I just realized this is the `mixer build upstream-format` one. That means it *also* has to do the finagling of `upstreamversion` before running in container.

`mixer build format-bump` is the one that needs the above 4 lines without the extra finagling.